README.md

```diff
@@ -56,7 +56,7 @@ import com.softwaremill.quicklens._
 
 case class Street(name: String)
 case class Address(street: Option[Street])
-case class Person(addresses: List[Address])
+case class Person(addresses: Seq[Address])
```

Comment:
I was worried when I first read the docs that you had to use `List`s, so I changed all the types to `Seq`.

Reply:
Ok, let's mention `Seq`s as the supported collections, but keep `List`s in the examples. That's the most common collection  :)